### ISSUE TITLE
Missing nuget version: Microsoft.Identity.Client.1.1.1-alpha0414

### ISSUE THREAD

**Issue by JohnHMP** [AzureAD/microsoft-authentication-library-for-dotnet#590]
UPDATE: Urgh, it appears this version is on the overnight nuget feed, not on nuget.org. I am not sure why our droid VSTS CI build is no longer able to see it (the overnight build feed is in the same nuget.config that the iOS build has), but it is not the issue I reported. This issue should be closed.

We have been piloting a new app that uses MSAL.NET for auth and the nuget package version we are using is 1.1.1-alpha0414. Unfortunately our VSTS CI Android builds are failing as that version can no longer be found, and indeed I cannot see that version listed on nuget at all.

We can of course update to the latest (at time of writing 1.1.4-preview0002) but that will break our dev cycle, upgrades to nugets and dependencies need to be planned etc.

Has that version been deleted?

**Comment by bgavrilMS** [comment]
Hi @JohnHMP - did you find the missing package?

**Comment by JohnHMP** [comment]
Yes, I found the version on the overnight feed. The issue I am having is that our VSTS CI build for our Droid app is not finding it. I *think* it is because it finds the package name on nuget.org so doesn't look at the overnight feed. I am testing a solution where I do a nuget restore from the overnight feed before anything else in the VSTS build configuration.

**Comment by JohnHMP** [comment]
@bgavrilMS actually, no that specific version isn't in the overnight feed anymore. I am guessing some housekeeping has taken place and only the last 30 overnight builds a retained.
I have done a compare between the build date of the 1.1.1 alpha dll and the release date of the 1.1.1 preview version on master, and there appears to be no code changes.
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/compare/master@%7B2018-01-06%7D...master@%7B2017-12-11%7D

I presume 1.1.1 preview would be the equivalent official nuget package to use instead of the head?

**Comment by bgavrilMS** [comment]
@JohnHMP - I don't know the history of MSAL as I joined the team only a few weeks ago. However, if the alpha is no longer available and you have to switch to the preview, you might as well move to the latest preview?

Otherwise, I would expect there to be no public surface changes between the alpha and the preview, maybe some bug fixes. 

@jennyf19 or @henrik-me may know more (they are on the PST timezone).

**Comment by JohnHMP** [comment]
@bgavrilMS normally I would agree, but if I can be sure of no code changes between the 2 builds then I don't have to interrupt this sprint. I can schedule the upgrade to the latest preview in a later sprint, however.
